perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -1063,6 +1071,11 @@ sub check_for_errors : Private {
         delete $field_errors{username};
     }
 
+    # if we're contributing as someone else then allow landline numbers
+    if ( $field_errors{username} && $c->stash->{contributing_as_another_user} && !$c->stash->{phone_may_be_mobile}) {
```

Comment:
If they've entered an invalid email address, this will let it through, because phone_may_be_mobile will be false.
(same check is in Update.pm too)